### PR TITLE
ignore deleted files

### PR DESCRIPTION
**Pull request by archerabi**

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import * as core from '@actions/core';
 import * as github from '@actions/github';
 import { CHECK_NAME, EXTENSIONS_TO_LINT } from './constants';
 import { eslint } from './eslint-cli';
-
+import { existsSync } from 'fs';
```
Comment (myfashionhub) [inline src/main.ts]: group with `path` import

Comment (archerabi) [reply src/main.ts]: don't follow this. Why should i group with path ?

Comment (myfashionhub) [reply src/main.ts]: they're system modules
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md

**Comment by archerabi** [comment]
thanks @myfashionhub

**Pull request merged**